mgmt-lambda/exceptions.ts

```diff
@@ -0,0 +1,26 @@
+export enum ErrorCode {
+  UnknownError = 'E1000',
+  AWSResourceNotFound = 'E2100',
+  AWSAccessDenied = 'E2200',
+  LambdaFunctionNotFound = 'E3100',
+  LambdaFunctionAssociationNotFound = 'E6100',
+  CloudFrontDistributionNotFound = 'E4100',
+  CacheBehaviorNotFound = 'E5100',
+  CacheBehaviorPatternNotDefined = 'E5200',
+  FunctionARNNotFound = 'E7100',
```

Comment:
Why don't we call this `LambdaFunctionARNNotFound`? Another question, Why don't we group all `LambdaFunction...` errors together, like `E3XXX`?

Reply:
There is no reason we don't call this `LambdaFunctionARNNotFound` other than it's too verbose. I tried to group by context first and then it kinda got randomized. The reason we implementing error codes on response has no use for grouping so probably it's not going to worth discussing, skipping this. If do you think it's worths the effort we should discuss tho. Skipping this for now.

Reply:
I understand that it is too verbose, but others are called `LambdaFunction`, only this one is called `Function`. Why are they different? I am fine with calling all of them `Function` or `LambdaFunction` (I like `LambdaFunction` better if it matters, verbose is good)